### PR TITLE
PWM Initialization

### PR DESCRIPTION
**Pull request by JuanLNH**
#3 
 Definition of done:

- [x] Initialize the PWM.
- [x] Add the PWM configuration to generate a signal with a frequency of 1 kHz. 
- [x] Unit test.

QA:
- [x] Behaves as expected. 

Connect terminals of osciloscope in the PT45 from board.
Osciloscope should meassure a 1KHz of frequency.

### CODE REVIEW

File: Code/Integrator_Project/src/hal_entry.c [hunk 1]
```diff
@@ -12,16 +12,19 @@ int adcData = 0;
 int percentage = 0;
 
 void hal_entry(void) {
+
```
Comment (IrvinRamiA) [inline Code/Integrator_Project/src/hal_entry.c]: EWS

File: Code/Integrator_Project/src/hal_entry.c [hunk 2]
```diff
@@ -12,16 +12,19 @@ int adcData = 0;
 int percentage = 0;
 
 void hal_entry(void) {
```
Comment (IrvinRamiA) [inline Code/Integrator_Project/src/hal_entry.c]:
```suggestion
 void hal_entry(void)
 {
```

File: Code/Integrator_Project/src/hal_entry.c [hunk 3]
```diff
@@ -12,16 +12,19 @@ int adcData = 0;
...
+
     while(1)
     {
         g_adc0.p_api->read(g_adc0.p_ctrl, ADC_REG_CHANNEL_0, &adcData);
         percentage = (adcData * 100) / 255;
     }
-
 }
 
 /*
```
Comment (IrvinRamiA) [inline Code/Integrator_Project/src/hal_entry.c]: Erase comment

File: Code/Integrator_Project/src/hal_entry.c (hunk 1 above)
Comment (JuanLNH) [reply Code/Integrator_Project/src/hal_entry.c]: Done!

**Review by IrvinRamiA — COMMENTED** [review]
LGTM

File: Code/Integrator_Project/src/hal_entry.c (hunk 2 above)
Comment (JuanLNH) [reply Code/Integrator_Project/src/hal_entry.c]: Done!

File: Code/Integrator_Project/src/hal_entry.c (hunk 3 above)
Comment (IrvinRamiA) [inline Code/Integrator_Project/src/hal_entry.c]: Erase comment

Comment (JuanLNH) [reply Code/Integrator_Project/src/hal_entry.c]: Done!

**Pull request merged**